src/tab.group.test.events.ts

```diff
@@ -1,24 +1,30 @@
 import { expect, fixture, html, oneEvent } from '@open-wc/testing';
 import { sendKeys } from '@web/test-runner-commands';
-import './tab.group.js';
-import './tab.js';
-import './tab.panel.js';
+import './tab-group.js';
+import './tab-group.tab.js';
+import './tab-group.panel.js';
 import { click } from './library/mouse.js';
 
 it('dispatches a "selected" event on click', async () => {
   const host = await fixture(html`
     <glide-core-tab-group>
-      <glide-core-tab panel="1" slot="nav">One</glide-core-tab>
-      <glide-core-tab panel="2" slot="nav">Two</glide-core-tab>
+      <glide-core-tab-group-tab
+        label="One"
+        panel="1"
+        slot="nav"
+      ></glide-core-tab-group-tab>
+      <glide-core-tab-group-tab
+        label="Two"
+        panel="2"
+        slot="nav"
+      ></glide-core-tab-group-tab>
 
-      <glide-core-tab-panel name="1">One</glide-core-tab-panel>
-      <glide-core-tab-panel name="1">Two</glide-core-tab-panel>
```

Comment:
Actually caught some validation issues of a matching tab + panel here and in a few other tests🎉